appveyor.yml

```diff
@@ -25,12 +25,12 @@ environment:
       OTP_VERSION: 20.1
       ERTS_VERSION: 9.1
       BUILD_STEP: build
-      TEST_STEP: ct
+      TEST_STEPS: ct,release
```

Comment:
I am a bit surprised that these are note rendered as 4 jobs:
* OTP 20.1 ct - artefacts being CT logs and node logs
* OTP 20.3 ct - artefacts being CT logs and node logs
* OTP 20.1 release - artefacts packages, only if smoke test passes
* OTP 20.3 release - artefacts packages, only if smoke test passes

If too slow, the 20.1 jobs may be deleted.

If too complex, they can be kept as is.

Reply:
On AppVeyor CI it is not possible to share build artifacts between jobs. Therefore, splitting the test steps into 2 job would result in an additional build/package cycle which adds quite some time. That is why I condensed it into a single job.